README.md

```diff
@@ -1,61 +1,86 @@
-# Cherrytwist - Challenge Management Platform 
-Welcome to CherryTwist - an open source platform for managing Challenges that allows you to: 
-
-* Manage the full lifecycle of innovation challenges, in a single, shared representation
-    * Challenges: Scope, publish, campaign
-    * Project portfolio: Initiate, (co)fund and govern projects
-    * Grants: (crowd)fund, award and govern, keep track of results and paper-trail
-* Community: Engage and grow your community
-    * Onboard stakeholders and talented / motivated contributors, increase diversity of contributions
-    * Enable exchange of capabilities, expertise and knowledge
-* Enable sharing & learning within and between communities
-* Self sovereign / decentralisation ready (web 3 under the hood)
-
-This repository is for coordination around the project. The project is developing rapidly, and we aim to get the basics right from the get go so please do share any feedback / observations / suggestions you have!
-
-#  What is the problem?
-Managing challenges is hard:
-* Many parties involved, different backgrounds
-* Fragmented best practices / community 
+
+<p align="center">
+  <a href="http://cherrytwist.org/" target="blank"><img src="https://cherrytwist.org/wp-content/uploads/2020/10/cherrytwist-2.png" width="240" alt="Cherrytwist Logo" /></a>
+</p>
+<p align="center"><i>Enabling society to collaborate. Building a better future, together.</i></p>
+
+
+# Cherrytwist: Challenge Management Platform 
+Cherrytwist is an open source platform for managing Challenges throughout their lifecycle. 
+
+## What is a Challenge? 
+A Challenge is any desired outcome whereby multiple parties need to coordinate their activities over a longer period to solve. 
+
+Challenges arise from all aspects of society: whether we’re talking about UN SDGs or corporates trying to innovate with purpose for a sustainable future, governments at any level engaging with their citizens, or a local community working together for a common interest.  
```

Comment:
Don't use acronyms on first occurrence. It's better to use United Nations (UN), Sustainable Develpment Goals (SDGs) etc to establish the context. You assume the reader has context otherwise, which is by no means a given.